entity/templates/src/main/java/package/web/rest/_EntityResource.java

```diff
@@ -40,15 +41,10 @@
 public class <%= entityClass %>Resource {
 
     private final Logger log = LoggerFactory.getLogger(<%= entityClass %>Resource.class);
-
-    @Inject
-    private <%= entityClass %>Repository <%= entityInstance %>Repository;<% if (dto == 'mapstruct') { %>
-
-    @Inject
-    private <%= entityClass %>Mapper <%= entityInstance %>Mapper;<% } %><% if (searchEngine == 'elasticsearch') { %>
-
-    @Inject
-    private <%= entityClass %>SearchRepository <%= entityInstance %>SearchRepository;<% } %>
+    <% var viaService = service != 'no'; -%>
```

Comment:
@jdubois I dont understand your question here? do you mean ending each line with a yo template tag and starting again in next line? if so I did this way to highlight the include, if you dont like it ill remove and make it in one block